pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -763,6 +791,17 @@ func TestBadNameDaemonSetDoesNothing(t *testing.T) {
 	syncAndValidateDaemonSets(t, manager, ds, podControl, 0, 0)
 }
 
+// DaemonSet with node selector, matching a node, should launch a pod on the node.
+func TestSelectorDaemonSetLaunchesPods(t *testing.T) {
```

Comment:
prefer to combine those two case into one test func, and it's a test cases (func) for nodeSelector.